### DISCUSSION TITLE
Unable to choose from list of possible directories when moving files

### DISCUSSION THREAD

**Discussion by kyrillh** [nvim-tree/nvim-tree.lua#2136] [Q&A]
Steps to reproduce: 
 - Mark a bunch of files for moving with m 
 - Initiate the move with bmv
 - In the prompt that appears I can start typing a folder name and <TAB> to get a list of folders matching the name. However, I am unable to choose from this list. Pressing the up and down arrow keys selects the first entry immediately. 

Is there a special way to choose from this list, is this a bug or is this perhaps due to conflicting plugins? 

I am new to neo-vim (and vim for that matter) so any tips would be greatly appreciated :)

**Comment by gegoune** [comment]
nvim-tree does not provide any completion so this is most likely coming from whatever completion plugin you are using. If you think this is an issue with nvim-tree please open an issue with minimal configuration reproduction.
